lib/ansible/content.rb

```diff
@@ -33,11 +33,13 @@ def self.fetch_plugin_galaxy_roles
     def self.consolidate_plugin_content(dir = PLUGIN_CONTENT_DIR)
       require "vmdb/plugins"
 
+      roles_dir = dir.join("roles")
       FileUtils.rm_rf(dir)
-      FileUtils.mkdir_p(dir)
+      FileUtils.mkdir_p(roles_dir)
 
       Vmdb::Plugins.ansible_content.each do |content|
-        FileUtils.cp_r(Dir.glob("#{content.path}/*"), dir)
+        new(content.path).fetch_galaxy_roles
+        FileUtils.cp_r(Dir.glob(content.path.join("roles/*/")), roles_dir)
```

Comment:
The trailing slash on the glob here is important to avoid copying the requirements.yml and .gitignore file.  All of the roles will be in subdirectories.

Reply:
Do we need anything outside of the roles directory?

Do we need to remove the copied over files from content.path after it's been copied?

Reply:
Nope, just the roles directory (for now - if we add support for collections later, then it will just be a code change here anyway).

> Do we need to remove the copied over files from content.path after it's been copied?

Not really necessary, though that does make me wonder if we have just extra data floating around. It was a `cp_r` before, so this part of the code didn't really change, so I think we can merge anyway, and treat that as a separate cleanup step afterwards.